src/ILVerify/src/ILVerify.csproj

```diff
@@ -1,19 +1,55 @@
-<Project Sdk="Microsoft.NET.Sdk">
-
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFrameworks>netcoreapp2.0;net46</TargetFrameworks>
     <PlatformTarget>AnyCPU</PlatformTarget>
     <CLSCompliant>false</CLSCompliant>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
   </PropertyGroup>
+  <PropertyGroup>   
+    <AssemblyName>ILVerify</AssemblyName>
+    <AssemblyKey></AssemblyKey>
+    <AssemblyOriginatorKeyFile>..\..\ILVerification\StrongNameKeys\ILVerify.snk</AssemblyOriginatorKeyFile>   
+  </PropertyGroup>
+  
+  <ItemGroup>
+    <Content Include="ilverify.runtimeconfig.json">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </Content>
+  </ItemGroup>
+
 
   <ItemGroup>
     <Compile Include="Program.cs" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="System.CommandLine" Version="0.1.0-e160909-1" />    
+    <PackageReference Include="System.CommandLine" Version="0.1.0-e160909-1" />
     <ProjectReference Include="..\..\ILVerification\src\ILVerification.csproj" />
   </ItemGroup>
+  <ItemGroup>
+    <Compile Include="..\..\Common\src\CommandLine\CommandLineException.cs">
+      <Link>CommandLine\CommandLineException.cs</Link>
+    </Compile>
+    <Compile Include="..\..\Common\src\CommandLine\CommandLineHelpers.cs">
+      <Link>CommandLine\CommandLineHelpers.cs</Link>
+    </Compile>
+    <Compile Include="..\..\Common\src\System\NotImplemented.cs">
+      <Link>System\NotImplemented.cs</Link>
+    </Compile>
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Include="..\..\ILVerification\StrongNameKeys\ILVerify.snk" Link="ILVerify.snk" />
+  </ItemGroup>
+
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.targets))\dir.targets" />
+
+  <PropertyGroup>
+    <RunCommand Condition="'$(OS)' == 'Windows_NT'">$(TargetDir)$(AssemblyName).exe</RunCommand>
+    <RunCommand Condition="'$(OS)' != 'Windows_NT'">$(TargetDir)$(AssemblyName)</RunCommand>
+    <RunArguments>$(StartArguments)</RunArguments>
```

Comment:
Are you able to debug ILVerify in Visual Studio with these changes? (E.g. open ILVerify.csproj in VS and verify that you can hit a breakpoint.)